Sources/Utils/Atomic.swift

```diff
@@ -32,6 +32,12 @@ public final class Atomic<Value> {
     public var value: Value {
         get { return withValue { $0 } }
         set(newValue) { swap(newValue) }
+        _modify {
```

Comment:
Was this addition intended?

Reply:
Well, it was, even though it's not related to deprecating iOS 9. Technically it only requires Swift 5.0, but I forgot to put it in on the migration PR. 😇 

Do you prefer that I add it on a separate PR?

Reply:
I'd suggest moving this to a separate PR. It's a _little big_ change to be introduced here.

Reply:
Fair enough, will do it 👍